app/src/main/res/values/dimens.xml

```diff
@@ -2,4 +2,8 @@
 <resources>
     <dimen name="standard_padding">16dp</dimen>
     <dimen name="small_padding">8dp</dimen>
+    <dimen name="medium_width">120dp</dimen>
```

Comment:
this seems okay for now but soon these names would be confusing. You might want to call it `medium_button_width`. Same for others below

Reply:
Not critical to change this now, but I'll leave it up to you

Reply:
Ok, just updated it